python/lsst/cp/verify/mergeResults.py

```diff
@@ -84,101 +145,167 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         outputs = self.run(**inputs)
         butlerQC.put(outputs, outputRefs)
 
-    def run(self, inputStats, camera, inputDims):
+    def run(self, inputStats, inputDims, camera=None, inputResults=None, inputMatrix=None,):
         """Merge statistics.
 
         Parameters
         ----------
```

Comment:
If I'm reading the diff correctly you removed the Parameters instead of fixing them?

Reply:
No, I forgot to update this docstring.  Should be fixed now.